MapboxNavigation/BottomBannerViewLayout.swift

```diff
@@ -95,9 +94,6 @@ extension BottomBannerViewController {
     }
     
     fileprivate func setupVerticalRegularLayout(_ c: inout [NSLayoutConstraint]) {
-        let size = bottomBannerView.heightAnchor.constraint(equalToConstant: 80)
```

Comment:
Since `bottomBannerView` is added inside `BottomBannerViewController` which is in turn embedded into `bottomBannerContainerView` setting height constraint has no effect here.